src/App.js

```diff
@@ -1,13 +1,74 @@
 import React from 'react';
 
+import './components/TodoComponents/Todo.css'
+
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+
+const data = [
+  {
+   task: 'Pick up milk',
+   id: Date.now(),
+   completed: false
+  }
+];
+
+
+
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state ={
+      data: data,
+      task: '',
+      id: '',
+      completed: ''
+    };
+  }
+
+  addTodo = e => {
+    e.preventDefault();
+    const newTodo = {
```

Comment:
Since the new todo set's up and id and completed will always be Date.now and false you won't need to keep track of it in state